### PR TITLE
Review Model & Controller Created

### PR DESCRIPTION
**Pull request by KKennedyCodes**
Created Model and Controller
Set up validations and index and create controller methods.

### CODE REVIEW

File: app/models/review.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class Review < ApplicationRecord
+  belongs_to :product
+  
+  validates :product_id, presence: true
+  validates :rating, numericality: { only_integer: true, greater_than: 0, less_than: 6 }
```
Comment (raisahv) [inline app/models/review.rb]:
```suggestion
  validates :rating, presence: true, numericality: { only_integer: true, greater_than: 0, less_than: 6 }
```

**Comment by raisahv** [comment]
looks great!

**Review by NatalieTapias — COMMENTED** [review]
LGTM

**Review by dtingg — APPROVED** [review]
Looks good

**Pull request merged**